pyproject.toml

```diff
@@ -213,3 +210,19 @@ ignore_errors = true
 
 # Change the below to invalidate dependency cache in CI
 # CACHE-TAG: 1
+
+[tool.ruff]
+lint.select = ["E", "F", "B"]
+lint.ignore = [
+    "F541",
+    "B008",
+    "B904",
+    "B905",
+    "B019",
+    "E731",
+    "E741",
+]
+line-length = 80
+indent-width = 4
+target-version = "py310"
```

Comment:
No need for this, because `ruff` looks at `requires-python` that we already have above